### ISSUE TITLE
[Feature request] Allow trigger quickadd in context menu

### ISSUE THREAD

**Issue by Quorafind** [chhoumann/quickadd#27]
Found that a new api for context menu is released in ver 0.12.6, would you want to add support for it?

**Comment by chhoumann** [comment]
Yes! This is the plan. I have been considering adding support for both the `file-menu` and the new `context-menu`. It would work somewhat like the 'Add command for choice' works, where you can add certain choices (and maybe macros) to the menus.

**Comment by connectdotz** [comment]
hi, @chhoumann first of all, thanks for this wonderful plugin! 

Just wondering what is the plan for the requested feature. Would love to see this supported so we can add actions to, for example, text selections, in which the slash-command doesn't quite work (it replaces the selection with `/`)...
